_includes/help/nav_sidenav.html

```diff
@@ -1,20 +1,44 @@
-<nav class="sm-col-right sm-col-4 sm-show">
-  <ul class="list-reset pt2">
-    {% for question in related_questions %}
-      {% if question.url == page.url %}
-        {% assign _index = forloop.index | modulo:7 %}
-        {% if _index == 0 %}
-          {% assign _index = 1 %}
+<div class="grid-col grid-col-3 grid-offset-2">
+  {% include search.html size="small" %}
+  <nav class="margin-y-4 sidenav">
+    <ul class="usa-accordion usa-sidenav">
+      {% for subpage in site.help_pages %}
+      <li class="usa-sidenav__item__empty">
+        {% assign subpage_slug = subpage | slugify %}
+        {% assign related_questions = site.help | where_exp: "item", "item.url contains subpage_slug" | sort: 'order' %}
+        {% capture is_initially_expanded %}
+        {% if page.url contains subpage_slug %}true{% else %}false
         {% endif %}
-      {% endif %}
-      <a class="block h6 nav-sidenav-item {% if question.url == page.url %}active{% endif %}" href="{{ question.url | prepend: site.baseurl }}">
-        <li class="p2 {% unless forloop.last %}border-bottom{% endunless %}">
-          {% assign url_size = question.url | size | minus: 1 %}
-          {% assign page_title = question.url | replace: "/", "." | slice: 1, url_size %}
-          {% capture page_title %}{% t page_title %}{% endcapture %}
-          {{ page_title | markdownify }}
-        </li>
-      </a>
-    {% endfor %}
-  </ul>
-</nav>
+        {% endcapture %}
+        {% capture is_initially_hidden %}
+        {% if page.url contains subpage_slug %}false{% else %}true
+        {% endif %}
+        {% endcapture %}
+      <li class="usa-sidenav__item usa-parent {% if is_initially_expanded %}usa-current{% endif %}"
```

Comment:
I'm seeing both "Changing settings" and "Trouble signing in?" headings appearing as "current", where I'd expect it should only have the latter as current.

At a glance, I think it may be that assigning using `{% capture %}`...

1. Assigns as text, so this would probably need to be:
```suggestion
      <li class="usa-sidenav__item usa-parent {% if is_initially_expanded == 'true' %}usa-current{% endif %}"
```
2. Includes extra whitespace, so the capture would need to be collapsed to a single line

Another option could be to use `assign` instead of capture to assign as a boolean.

```
{% assign is_initially_expanded = page.url contains subpage_slug %}
```